packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -205,7 +205,9 @@ class Checkout extends Component<
             }), extensionService.loadExtensions()]);
 
             const providers = data.getConfig()?.checkoutSettings?.remoteCheckoutProviders || [];
-            const supportedProviders = getSupportedMethodIds(providers);
+            const features = data.getConfig()?.checkoutSettings?.features || {};
+
+            const supportedProviders = getSupportedMethodIds(providers, features);
```

Comment:
Can we use this here https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/app/common/utility/isExperimentEnabled.ts#L3

Reply:
This function return `true` if an exp is `undefined` by default. It is not what we are expected.

Reply:
This is because we have had issues when an experiment is rolled out completely, say in a few weeks and a merchant forks checkout-js to create custom checkout in the meantime it causes us to never be able to remove the experiment. 
Hence this is the new approach to roll out experiments so we can successfully remove experiments.

Reply:
Thanks, that is a good point. I will keep it in mind and add changes related to it.